### PR TITLE
visual fix for bridge pipe and color adapter

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
fix bridge pipe and color adapter connector pipes from being messed up

## Why It's Good For The Game
visual fix

## Changelog
:cl:
fix: visual fix for bridge pipe and color adapter
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
why is this needed? just cause layer/color adapters only have one layer? or what

**Comment by Ghilker** [comment]
No this caused the added overlay pipe that show the connection to not work properly, like this 
unknown

File: tgstation.dme [hunk 1]
```diff
@@ -3061,8 +3061,8 @@
 #include "code\modules\reagents\chemistry\reagents\reaction_agents_reagents.dm"
 #include "code\modules\reagents\chemistry\reagents\toxin_reagents.dm"
 #include "code\modules\reagents\chemistry\reagents\impure_reagents\impure_medicine_reagents.dm"
-#include "code\modules\reagents\chemistry\reagents\unique\eigenstasium.dm"
 #include "code\modules\reagents\chemistry\reagents\impure_reagents\impure_toxin_reagents.dm"
+#include "code\modules\reagents\chemistry\reagents\unique\eigenstasium.dm"
```
Comment (ninjanomnom) [inline tgstation.dme:3065]: Is this change supposed to be here? This looks like what's on master already

Comment (Ghilker) [reply tgstation.dme]: fixed, thanks

**Comment by LemonInTheDark** [comment]
Ah I see

**Pull request merged**